trie/database.go

```diff
@@ -70,6 +70,10 @@ var (
 type Database struct {
 	diskdb ethdb.KeyValueStore // Persistent storage for matured trie nodes
 
+	// zktrie related stuff
+	Zktrie     bool
```

Comment:
`config.Zktrie` is not be used.

Reply:
It is used in stateDb for detouring some action when chain config has enabled zktrie